src/git/Repository.ts

```diff
@@ -229,11 +229,11 @@ export class Repository {
     }
 
     public static getLocalOriginUrl(repoName: string, rootDir: string): Promise<string> {
-        return new Promise<string>((resolve, reject) => {
+        return new Promise<string>((resolve) => {
             simpleGit(rootDir).remote(['get-url', 'origin'], (err, result: string) => {
                 if (err) {
-                    Global.isVerbose() && console.log(`[${repoName}]:`, ': git remote get-url failed!\n', err);
-                    reject(err);
+                    console.log(`[${repoName}]:`, 'git remote get-url failed! Has the root parent repository the remote added as "origin"?\n', err);
```

Comment:
The repos are cloned as expected, but
the output looks a bit strange if the origin is wrong/missing.

[main]: will clone the latest HEAD of remote branch master, depth -1, because useSnapshot is true.
[cplace-paw]: will clone the latest HEAD of remote branch master, depth -1, because useSnapshot is true.
Fehler: Remote-Repository 'origin' nicht gefunden

[main]: git remote get-url failed! Has the root parent repository the remote added as "origin"?
 Fehler: Remote-Repository 'origin' nicht gefunden

Fehler: Remote-Repository 'origin' nicht gefunden

[cplace-paw]: git remote get-url failed! Has the root parent repository the remote added as "origin"?
 Fehler: Remote-Repository 'origin' nicht gefunden
Could you try to suppress the error messages or check it in some other way?

Reply:
As discussed: part of the message is from git. IMHO this fix is for a an edge case like on the IDP so I think it's fine keeping the message - which would also help in case someone has checked out the repo with https and another origin and wonders why its not working.